### ISSUE TITLE
[enhancement] Current playing chapter in bold font

### ISSUE THREAD

**Issue by plissken1138** [husudosu/mpv-remote-app#15]
Hope this is still active development.

The title says it all, would be nice enhancement and I'd guess it's not too tricky to implement.

**Comment by husudosu** [comment]
Yeah, the project is still under development. I'm busy right now with work, but I'll implement it sometime. Of course if you or someone wants to do a PR and I'll accept it.

**Comment by husudosu** [comment]
Hey! 
It's been a while, I've implemented your feature request. If you experience any issue let me know!

https://github.com/husudosu/mpv-remote-app/releases/tag/1.0.10
If you using IzzyOnDroid repo the next version gonna be available around 20:00-21:00 (CET)

**Comment by plissken1138** [comment]
> Hey! It's been a while, I've implemented your feature request. If you experience any issue let me know!
> 

It always marks the first chapter, regardless of what chapter is playing or is chosen. Selecting chapters works.


Huawei P9, Android7

**Comment by husudosu** [comment]
Try to update your MPV version, by some reasons the currentChapter response from MPV only working on  the most recent version. I'll create a workaround for that.

**Comment by plissken1138** [comment]
> Try to update your MPV version, by some reasons the currentChapter response from MPV only working on the most recent version. I'll create a workaround for that.

it's always latest shinchiro daily build.

**Comment by husudosu** [comment]
Don't know then, for me it's working properly. I have to investigate
